lib/form_data.js

```diff
@@ -294,7 +295,8 @@ FormData.prototype.submit = function(params, cb) {
     // TODO: Add chunked encoding when no length (if err)
 
     // add content length
-    request.setHeader('Content-Length', length);
+    if (length)
+      request.setHeader('Content-Length', length);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'request'.](https://www.codacy.com/app/form-data/form-data/file/875565507/issues/source?bid=2805153&fileBranchId=2805172#l299)